app/controllers/provider/signups_controller.rb

```diff
@@ -62,6 +53,13 @@ def create
 
   protected
 
+  def signup_params
+    attributes = params.fetch(:account, {})
```

Comment:
Does it make sense using this action with this param missing? How about the following? (I haven't tested it BTW.)

```suggestion
    account_params = params.require(:account).merge(sample_data: true)
    user_params = account_params.delete(:user).merge(signup_type: :new_signup, username: 'admin')
```

Reply:
require raises an error and fetch would just return errors of "required params missing" which happens if no param is sent at all. I am a bit scared about doing that because it is changing behavior and I already had a bad experience trying this for another endpoint.

I will try though and I will see if it is a good idea or too risky :thinking:

Reply:
I've tested it, and without this change, it raises an ugly error trying to render the view xD
![image](https://user-images.githubusercontent.com/11318903/72739511-1028b500-3ba4-11ea-8485-b9d283f6d3e3.png)


So you are completely right on this one! Great catch!